app-modules/authorization/database/factories/RoleGroupFactory.php

```diff
@@ -36,17 +36,19 @@
 
 namespace Assist\Authorization\Database\Factories;
 
+use Assist\Authorization\Models\RoleGroup;
 use Illuminate\Database\Eloquent\Factories\Factory;
 
 /**
- * @extends \Illuminate\Database\Eloquent\Factories\Factory<\Assist\Authorization\Models\RoleGroup>
+ * @extends Factory<RoleGroup>
  */
 class RoleGroupFactory extends Factory
 {
     public function definition(): array
     {
         return [
-            'name' => fake()->catchPhrase,
+            'name' => fake()->word(),
```

Comment:
Why the change from `catchPhrase` to `word()` I feel like you and @dgoetzit keep swapping it each time y'all are in this file XD

Reply:
I changed it because it kept telling me the method doesn't exist. I double checked the docs and it does, and it works, so I'll put it back.